advanced_html_css/accessibility/the_web_content_accessibility_guidelines_wcag.md

```diff
@@ -1,6 +1,6 @@
 ### Introduction
 
-After the previous lesson, you should now have a better understanding of how important web accessibility can be for certain users, and how beneficial it can be for *all* users. But how do we know not only what should be a11y friendly, but how to make those things a11y friendly? Well, there are many sources available to tell us the what and how, those who actually rely on accessibility being one of the best sources we can consult.
+After the previous lesson, you should now have a better understanding of how important web accessibility can be for certain users, and how beneficial it can be for _all_ users. But how do we know not only what should be a11y friendly, but how to make those things a11y friendly? Well, there are many sources available to tell us the what and how, those who actually rely on accessibility being one of the best sources we can consult.
```

Comment:
For the markdown we use asterisk. Not the _ _. If you could revert it back to the asteriks that would be great! You can check it out in this part of the [style guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md#markdown-styling) if your curious.